src/Styleguide/Components/Stepper.tsx

```diff
@@ -0,0 +1,97 @@
+import { color, space } from "@artsy/palette"
+import React from "react"
+import styled from "styled-components"
+import { Tab, Tabs, TabsProps } from "Styleguide/Components/Tabs"
+import { Flex } from "Styleguide/Elements/Flex"
+
+interface StepperProps extends TabsProps {
+  initialTabIndex: number
+
+  /* the step user currently is at */
+  currentStepIndex: number
+}
+
+const transformTabBtn = (
+  element: JSX.Element,
+  tabIndex: number,
+  props: any
+): JSX.Element => {
+  const { currentStepIndex } = props
+  if (tabIndex > currentStepIndex) {
+    // don't allow users to jump ahead
+    const newElement = React.cloneElement(element, {
+      ...element.props,
+      onClick: () => {
+        null
+      },
+    })
+    return newElement
+  } else if (currentStepIndex && tabIndex < currentStepIndex) {
+    return (
+      <Flex>
+        <CheckMarkWrapper>
+          <CheckMark />
+        </CheckMarkWrapper>
+        {element}
+        <div /> {/* hack for getting rid of last-child in Tabs.tsx */}
+      </Flex>
+    )
+  } else return element
+}
+
+export const Stepper = (props: StepperProps) => {
+  return (
+    <Tabs
+      separator={
+        <ChevronWrapper>
+          <Chevron />
+        </ChevronWrapper>
+      }
+      transformTabBtn={transformTabBtn}
+      {...props}
+    />
+  )
+}
+
+export const Step = props => <Tab {...props} />
+
+const ChevronWrapper = styled.span`
+  margin: 0 ${space(2)}px;
+  line-height: normal;
+`
+
+const CheckMarkWrapper = styled.span`
+  margin-right: ${space(1)}px;
+  line-height: normal;
+`
+
+const CheckMark = () => (
```

Comment:
@sepans I would say move those to `Assets/Icons/`. That's where all our svg components are

Reply:
thanks @l2succes. moved it in https://github.com/artsy/reaction/pull/1049/commits/dc4310a04b9eb1b467b2d96e464d9efbc85e248c. lmk if you have comments about the conventions you use (e.g. passing `color` vs having multiple components for different columns).